src/rules/at-function-named-arguments/README.md

```diff
@@ -0,0 +1,115 @@
+# at-function-named-arguments
+
+Require named parameters in SCSS function call rule.
+
+```scss
+animation: animation($duration: 250ms) {
+//                 ↑
+// Require or disallow this
+```
+
+## Options
+
+`string`: `"always"|"never"|"always-multiple-arguments"`
+
+### `always`
+
+The following patterns are considered warnings:
+
+```scss
+.foo {
+  animation: animation(250ms, 100ms, infinite);
+} 
+```
+
+```scss
+.foo {
+  animation: animation(250ms);
+} 
+```
+
+```scss
+.foo {
+  border: reset($value: 20, 'bar', $color: #FFF);
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```scss
+.foo {
+  animation: animation($duration: 250ms);
+}
+```
+
+```scss
+.foo {
+  animation: animation($duration: 250ms, $delay: 100ms, $direction: infinite);
+}
+```
+
+### `never`
+
+The following patterns are considered warnings:
+
+```scss
+.foo {
+  border: reset($value: 20);
+}
+```
+
+```scss
+.foo {
+  animation: animation($duration: 250ms, $delay: 100ms, $direction: infinite);
+}
+```
+
+```scss
+.foo {
+  border: reset($value: 20, 'bar', $color: #FFF);
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```scss
+.foo {
+  animation: animation(250ms, 100ms, infinite);
+} 
+```
+
+### `always-multiple-arguments`
+
+The following patterns are considered warnings:
+
+```scss
+.foo {
+  animation: animation(250ms, 100ms, infinite);
+}
+```
+
+```scss
+.foo {
+  border: reset($value: 20, 'bar', $color: #FFF);
+}
+```
+
+```scss
+.foo {
+  animation: animation($duration: 250ms);
```

Comment:
Why is this considered a warning when the rule is set to `always-multiple-arguments`?

Reply:
That’s is a good point. Would `only-multiple-arguments` make sense?

Reply:
I did a search in `stylelint` repo, but I didn't see `only-` prefix used in any of the rules, so maybe we should also try to keep things consistent.

I think that we can keep the name as it is right now, or then do something like `"always"` with a secondary option `{ except: ["single-argument"] }`, but I think that that might just be more confusing.

I'll need to sleep the night first to be able to think clearly :)

Reply:
Hey, sorry I'm just getting back to this. I agree it's important to keep things consistent. 

If we add the secondary `{ "except": ["single-argument"] }` option, would using it along with `"never"` mean that single arguments have to be named?

Looking at `stylelint`'s guidelines another approach can be `{ "ignoreSingleArgument": true }` which basically would avoid checking single arguments for either `"never"` or `"always"`. I'm not sure that I like this either, but it's more clear then `"except"` in my opinion.

What do you think?

Reply:
Using `"ignore"` could work, so how about `{ "ignore": ["single-argument"] }`? That style would also be extendable in the future.

Reply:
That sounds good. I’ll make this change.